shifthelper/conditions.py

```diff
@@ -274,17 +274,14 @@ def is_no_shift_at_the_moment():
 
 
 @log_call_and_result
-def is_nobody_awake():
-    '''Parker not Awake'''
-    awake = {}
+def is_nobody_ready_for_shutdown():
```

Comment:
The method name and the return value `not ready_ready_for_shutdown` suggest a double negative. To my puny human mind `is_anybody_ready_for_shutdown` can be converted more quickly into the corresponding logical formula. But thats just nitpicky stuff. Not important.

Reply:
I agree, this is complicated.

There is however an "interface" defined here. All these conditions can be listed in a "list of conditions", e.g. like here: 
https://github.com/fact-project/shifthelper/blob/a94949748acd2aa5221d573a568d2346437f4254/shifthelper/__main__.py#L216-L223

If all conditions evaluate to True, then we emit a call. So we are not really free here, if we want to stick to these condition lists. Maybe this is a hint that these condition lists were not a good idea. What do you think? 

However, removing these condition lists for something else is a larger modification ... so, we should have a different PR for that.

Reply:
> If all conditions evaluate to True, then we emit a call. So we are not really free here, if we want to stick to these condition lists. Maybe this is a hint that these condition lists were not a good idea. What do you think?

Ah cool. Was not aware of that. Having these lists seems very reasonable to me. 
In theory one could put a `not` in front of these conditions in the list right? Might make it a bit more readable for me personally.

Also, if one concatenates all those lists and ends up in  [NNF](https://de.wikipedia.org/wiki/Negationsnormalform) or [DNF](https://de.wikipedia.org/wiki/Disjunktive_Normalform),  that would certainly be interesting from an academic standpoint (theory of control systems and all that), but probably not of much use for anything else.  

In any case, I propose to leave it like that for now. I Seems very reasonable to me.

Reply:
perfect ... thanks for this very interesting feedback.